npm/private/npm_translate_lock.bzl

```diff
@@ -70,13 +73,12 @@ _ATTRS = {
     "repositories_bzl_filename": attr.string(default = DEFAULT_REPOSITORIES_BZL_FILENAME),
     "root_package": attr.string(default = DEFAULT_ROOT_PACKAGE),
     "update_pnpm_lock": attr.bool(),
-    "update_pnpm_lock_node_toolchain_prefix": attr.string(),
     "use_home_npmrc": attr.bool(),
+    "use_starlark_yaml_parser": attr.bool(),
```

Comment:
Is there a reason we have to keep it? Can we mark it as deprecated to delete in 2.0?

Reply:
See PR summary and docstring for this attribute for reasoning. We could remove this in the future once the fix for https://github.com/pnpm/pnpm/issues/5414 lands and settles but there is no rush since its a simple logic branch to maintain.